README.rst

```diff
@@ -1,19 +1,30 @@
 ======================================================
-Python Analytics for Snowplow 
+Python Analytics for Snowplow
 ======================================================
 .. image:: https://img.shields.io/static/v1?style=flat&label=Snowplow&message=Early%20Release&color=014477&labelColor=9ba0aa&logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAMAAAAoLQ9TAAAAeFBMVEVMaXGXANeYANeXANZbAJmXANeUANSQAM+XANeMAMpaAJhZAJeZANiXANaXANaOAM2WANVnAKWXANZ9ALtmAKVaAJmXANZaAJlXAJZdAJxaAJlZAJdbAJlbAJmQAM+UANKZANhhAJ+EAL+BAL9oAKZnAKVjAKF1ALNBd8J1AAAAKHRSTlMAa1hWXyteBTQJIEwRgUh2JjJon21wcBgNfmc+JlOBQjwezWF2l5dXzkW3/wAAAHpJREFUeNokhQOCA1EAxTL85hi7dXv/E5YPCYBq5DeN4pcqV1XbtW/xTVMIMAZE0cBHEaZhBmIQwCFofeprPUHqjmD/+7peztd62dWQRkvrQayXkn01f/gWp2CrxfjY7rcZ5V7DEMDQgmEozFpZqLUYDsNwOqbnMLwPAJEwCopZxKttAAAAAElFTkSuQmCC
     :alt: Early Release
-    :target: https://github.com/snowplow/snowplow/wiki/Tracker-Maintenance-Classification
-.. image:: https://travis-ci.org/snowplow/snowplow-python-tracker.svg?branch=master
+    :target: https://docs.snowplowanalytics.com/docs/collecting-data/collecting-from-own-applications/tracker-maintenance-classification/
+.. image:: https://img.shields.io/github/workflow/status/snowplow/snowplow-python-tracker/build
```

Comment:
Whilst we usually use shields.io for badges, I think preferring GitHubs own badge for builds is better: https://github.com/snowplow/snowplow-python-tracker/actions/workflows/ci.yml/badge.svg